docs/rancher.md

```diff
@@ -0,0 +1,320 @@
+How to use TopoLVM with Rancher/RKE
+===================================
+
+This document is a brief introduction of how to use TopoLVM on [Rancher/RKE](https://rancher.com/docs/rke/latest/en/).
+
+4 VMs will be used in this example. Each VM will be configured to have the following role:
+
+| Hostname  | Machine Type    | Role              | Requirement             |
+| --------- | --------------- | ----------------- | ----------------------- |
+| `rancher` | `n1-standard-2` | Rancher Server    | Allow HTTP/HTTPS        |
+| `master`  | `n1-standard-2` | Kubernetes Master |                         |
+| `worker1` | `n1-standard-2` | Kubernetes Worker | Mount 1 SSD for TopoLVM |
+| `worker2` | `n1-standard-2` | Kubernetes Worker | Mount 1 SSD for TopoLVM |
+
+## 1. Run Rancher Server
+
+### Create GCP instance
+
+Create a GCP instance for Rancher Server.
+
+```bash
+ZONE=asia-northeast1-c
+gcloud compute instances create rancher \
+  --zone ${ZONE} \
```

Comment:
Can I omit the `--zone` option from `gcloud` if the `gcloud` command is configured properly?

Reply:
I think so but I'm not sure if I should change the commands as you said.

Reply:
OK. Please try and fix appropriately.

Reply:
Why do you think that should be changed?
There is only a little difference, but I prefer not to force users to update their settings(=zone).

Reply:
Why do you force users to use `asia-northeast1-c`?
This is an official guide. If the default zone is set properly, we should let users use the zone which the user wants, I think.

Reply:
I don't mean to force them to use `asia-northeast1-c`.
Which zone they use is optional, but I just wanted to give it with `--zone`.

Reply:
I don't say anything anymore.

Of course, my base documentation is bad too.
But I think it's not good as an official document.